packages/new-polymath-issuer/src/pages/DividendDetails/Presenter.tsx

```diff
@@ -465,9 +476,15 @@ export const Presenter = ({
           </CardPrimary>
         </GridRow.Col>
       </GridRow>
-      <Heading variant="h2" mt="l" mb="m">
-        Transactions
-      </Heading>
+      {loading ? (
+        <Box mt="l" mb="m">
+          <LoadingDots />
+        </Box>
+      ) : (
+        <Heading variant="h2" mt="l" mb="m">
+          Transactions
+        </Heading>
+      )}
```

Comment:
Why is there a loading for this? Shouldn't it hide the entire table instead of just the heading?